psd-web/app/models/team.rb

```
@@ -26,12 +26,7 @@ def add_user(user_id)
   end
 
   def self.all(options = {})
-    begin
-      self.data = Shared::Web::KeycloakClient.instance.all_teams
-    rescue StandardError => error
```

@tom-bell-softwire, can you remember why we were swallowing these errors?

---

No... at a guess, it might have been to handle the issue where the initial request on startup fails, but more likely it was just a first approach to guard against any problems with getting Keycloak permissions working properly.

---

If this fails, does the whole service fall over (I can't remember if `User.all` is used when setting/getting `current_user`)?

---

One of the things it was doing, was preventing the asset compilation failing because it can't reach keycloak 🤦 I've reverted it for now...